src/caveexpress/server/entities/Player.cpp

```diff
@@ -154,6 +158,8 @@ void Player::update (uint32_t deltaTime)
 {
 	IEntity::update(deltaTime);
 
+	sendTargetCaveAngle();
```

Comment:
doing this every frame is a waste of traffic. it would be better to send the position once the target cave is selected and do the whole arrow thing client side.

Reply:
Right, I don't know yet how to do it on client side, which files/classes would I need to edit?